src/main/java/calculator/service/separator/SeparatorHandler.java

```diff
@@ -0,0 +1,53 @@
+package calculator.service.separator;
+
+import calculator.domain.number.Numbers;
+import calculator.domain.separator.BasicSeparator;
+import calculator.domain.separator.CustomSeparator;
+
+public class SeparatorHandler {
+
+    private static final String PREFIX_CUSTOM_SEPARATOR = "//";
+    private static final String SUFFIX_CUSTOM_SEPARATOR = "\\\\n";
+    private final SeparatorService separatorService;
```

Comment:
```suggestion
    private static final String PREFIX_CUSTOM_SEPARATOR = "//";
    private static final String SUFFIX_CUSTOM_SEPARATOR = "\\\\n";
    
    private final SeparatorService separatorService;
```

상수와 필드를 개행하면 조금 더 보기 편할 것 같습니다!